### PR TITLE
modified gluster_cluster_options value to dict type

### PR DESCRIPTION
**Pull request by kesavanvt**
Update Documentation to use dict value for gluster_cluster_options

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ Create a GlusterFS volume and set specific options.
     gluster_cluster_replica_count: 2
 
     # variables to set specific volume options
-    gluster_cluster_options: 'performance.cache-size:256MB'
+    gluster_cluster_options: {'performance.cache-size':'256MB'}
```
Comment (pkesavap) [inline README.md:75]: is this change tested? with single quotes?

**Review by pkesavap — CHANGES_REQUESTED** [review]
Could you update the PR with error and details with reason?

**Review by gobindadas — APPROVED** [review]

**Comment by kesavanvt** [comment]
**Errror msg :**
`FAILED! => {"ansible_facts": {"discovered_interpreter_python": "/usr/bin/python"}, "changed": false, "msg": "argument options is of type <type 'str'> and we were unable to convert to dict: dictionary requested, could not parse JSON or key=value"}`

**Reason:**
the `options` parameter of `gluster_volume` module takes value of dict type and not a string type

File: README.md (hunk 1 above)
Comment (kesavanvt) [reply README.md:75]: yes, the changes have been tested with single quotes

**Comment by pkesavap** [comment]
Looks good to me

**Pull request merged**